ptp/protocol/management_tlvs.go

```diff
@@ -303,3 +328,34 @@ func ParentDataSetRequest() *Management {
 		},
 	}
 }
+
+// ClockAccuracyRequest prepares request packet for CLOCK_ACCURACY request
+func ClockAccuracyRequest() *Management {
```

Comment:
Does it have to be public?

Reply:
It doesn't *have* to, but we keep all other Request() methods public, just so someone using the library can quickly construct them without relying on our management client implementation.

Reply:
I guess the question is if anybody needs a seemingly internal method ClockAccuracyRequest() or they just gonna use ClockAccuracy().

Reply:
But no strong opinion, just curious

Reply:
Thant's exactly what I meant. `ClockAccuracy` is a method of our ManagementClient, which is fairly dumb and limited. If anyone wants to send ClockAccuracy request (or any other management request) with any other client/transport, they need this helpful method to construct the request.